src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/CrimsonIsleReputationHelper.kt

```diff
@@ -74,6 +78,10 @@ class CrimsonIsleReputationHelper(skyHanniMod: SkyHanniMod) {
         if (!dirty && display.isEmpty()) {
             dirty = true
         }
+        if (!dirty && lastUpdate.passedSince() > 3.seconds) {
```

Comment:
Why do we redraw every 3 seconds? for the config value? see the property/value change detection instead.